src/main/java/Barbie.java

```diff
@@ -0,0 +1,182 @@
+import types.Deadlines;
+import types.Party;
+import types.Task;
+import types.Todo;
+import exceptions.*;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.*;
+
+
+public class Barbie {
+
+    enum Command {
+        MARK,
+        UNMARK,
+        DEL,
```

Comment:
I like that you have created an enumeration for the commands but perhaps it might be better to use the full word naming for the enum value of DELETE instead of DEL.